packages/components/src/internal/components/domainproperties/models.tsx

```diff
@@ -425,7 +425,7 @@ export class DomainDesign
         if (appPropertiesOnly) {
             columns = removeNonAppProperties(columns);
         }
-        if (domainKindName !== 'List') {
+        if (domainKindName !== 'VarList' && domainKindName !== 'IntList') {
```

Comment:
Previously we'd both verified that 'domainKindName' was not populating accurately for lists in particular, which was why this bug was kicked forward.

Reply:
It looks like these strings for `VarList` and `IntList` are used in a couple of places in the ui-components package. It might be time to add a constant for them in say `domainproperties/list/constants.ts` and use them in the various places.

Reply:
Done